readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -30,7 +30,7 @@ class TestLocalEnvironment(TestCase):
     def setUp(self):
         self.project = Project.objects.get(slug='pip')
         self.version = Version(slug='foo', verbose_name='foobar')
-        self.project.versions.add(self.version)
+        self.project.versions.add(self.version, bulk=False)
```

Comment:
What is the significance of this change? Did something change here that warrants a test change?